build-support/build-opts.js

```diff
@@ -0,0 +1,66 @@
+const execSync = require('sync-exec')
+const { env } = process
+const production = !!env.production
+const src = './src'
+const dest = env.dest ? env.dest : './dist'
+const noGlobals = !!env.noGlobals // don't generate globals bundle
+const noMap = !!env.noMap // don't generate .map files
+const codap = !!env.codap // include CODAP-specific modifications
+const assetsSrc = codap ? src + '/assets/img/*.*' : src + '/assets/**/*.*'
+const assetsDst = codap ? dest + '/img/' : dest
+
+const commit = execSync('git log -1 --pretty=format:"%H"').stdout
```

Comment:
```suggestion
const commitHash = execSync('git log -1 --pretty=format:"%H"').stdout
```
For clarity, as it wasn't immediately obvious to me.